api/app/tests/apps/signals/test_api_v1_endpoints.py

```diff
@@ -399,7 +408,82 @@ def test_create_initial(self):
         self.assertEqual(response_json['location']['created_by'], self.superuser.email)
         self.assertEqual(response_json['category']['created_by'], self.superuser.email)
 
```

Comment:
We want to guarantee that signals can be created when there is no address available, only the coordinates. So please add a test to that effect.

Reply:
added